app/src/main/java/io/pslab/fragment/InstrumentsFragment.java

```diff
@@ -161,7 +166,8 @@ protected Void doInBackground(Void... params) {
                     R.string.compass_description,
                     R.string.gyroscope_description,
                     R.string.thermometer_desc,
-                    R.string.robotic_arm_descriptoin
```

Comment:
Correct this one's spelling along with this

Reply:
Sorry for such blunders!. I type fast and then such mistakes get out of my attention. :joy: